protected/models/Address.php

```diff
@@ -66,6 +66,7 @@ public function rules()
         return array(
             array('address1, city, county', 'length', 'max' => 255),
             array('address1, city, county', 'required', 'on' => 'manage_practice'),
+            array('email','required', 'on'=>'self_register'),
```

Comment:
We keep debating with CERA as to whether or not this is mandatory. I wouldn't be surprised if they ask for it not to be mandatory again.

Reply:
We have a hard email saying what they want. For now that is gospel